public/view_users.php

```diff
@@ -24,23 +24,23 @@
         <table>
             <thead>
                 <tr>
-                    <th>#</th>
                     <th>First Name</th>
                     <th>Last Name</th>
                     <th>Email Address</th>
-                    <th>Hashed Password</th>
+                    <th>Instructor</th>
+                    <th>TA</th>
                     <th>Student ID</th>
                     <th>Created At</th>
                 </tr>
             </thead>
             <tbody>
         <?php foreach ($result as $row) { ?>
             <tr>
-                <td><?php echo escape($row->id); ?></td>
                 <td><?php echo escape($row->first_name); ?></td>
                 <td><?php echo escape($row->last_name); ?></td>
                 <td><?php echo escape($row->email); ?></td>
-                <td><?php echo escape($row->password_digest); ?></td>
+                <td><?php echo ( $row->is_instructor == true ? "yes" : "no"); ?></td>
```

Comment:
`echo ( $row->is_instructor ? "yes" : "no");`

don't think you need the == true

Reply:
done